### PR TITLE
#5783 [Manager] Action icons are not correctly rendered in some tables

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #issue_number)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/broker/BrokerNodeTable.tsx [hunk 1]
```diff
@@ -92,6 +91,9 @@ const BrokerNodeTable: React.FC<BrokerNodeTableProps> = ({
           {
             icon: 'visibility',
             tooltip: 'View node',
+            iconProps: {
+              color: 'action',
```
Comment (konekoya) [inline ohara-manager/client/src/components/broker/BrokerNodeTable.tsx:95]: Color consistency

**Comment by konekoya** [comment]
@saivirtue Could you help me test this PR? This now works locally on my machine.

**Review by saivirtue — APPROVED** [review]
Looks great!
Thanks for the quick fix 👍

**Review by eechih — APPROVED** [review]

**Comment by konekoya** [comment]
retry manager-it

**Comment by konekoya** [comment]
retry

File: ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts [hunk 2]
```diff
@@ -205,7 +205,9 @@ describe('Create Workspace', () => {
   });
 
   context('Volume', () => {
-    it('should able to add a volume', () => {
+    // FIXME: This test is currently failing due to Backend API issue
+    // Re-enable this after Backend issue is resolved
+    it.skip('should able to add a volume', () => {
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts:210]: @jackyoh do you have an issue tracking this? We can leave an issue number here so this comment won't be stale or forgetten

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts (hunk 2 above)
Comment (jackyoh) [reply ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts:210]: 在 issue #5798

Comment (konekoya) [reply ohara-manager/client/cypress/it/appBar/createWorkspace.test.ts:210]: No problem, I add update the comment in another PR as this PR has just passed all the tests...

**Pull request merged**